frontend/appflowy_flutter/lib/plugins/database/tab_bar/tab_bar_view.dart

```diff
@@ -371,6 +389,13 @@ class DatabasePluginWidgetBuilderSize {
   final double verticalPadding;
 }
 
+class DatabaseWidgetPaddingWithMaxWidth {
+  DatabaseWidgetPaddingWithMaxWidth({
+    this.left = 0.0,
+  });
+  final double left;
+}
+
```

Comment:
Can we reuse the DatabasePluginWidgetBuilderSize class instead of creating a new one?